osbuild/monitor.py

```diff
@@ -102,6 +102,13 @@ def begin(self, pipeline):
         self.out.write(f"Pipeline {pipeline.name}: {pipeline.id}")
```

Comment:
(commit message ` monitor/log: show build root and runner`)

Who told the `pipelie`? We need to find the liar!

Reply:
The cake is a lie!